staticd/static_vty.c

```diff
@@ -500,7 +500,7 @@ DEFPY_YANG(ip_route_blackhole,
       "Table to configure\n"
       "The table number to configure\n")
 {
-	if (table_str && vrf && !vrf_is_backend_netns()) {
```

Comment:
yea taking out this vrf check is the issue I believe.

Reply:
and add it to the other checks too i suspect.

Reply:
Please modify the check, in the NB backend API static_path_list_tableid_modify() and remove the checks from the static_vty.c file